resources/user-guides/attach-data.md

```diff
@@ -0,0 +1,113 @@
+# Using the Attach Data Rules Action
+
+The Attach Data rule action was added in the `Mobile Core v2.1.8` Launch extension. Attach Data is powerful and complex, and is recommended for users with advanced use cases.
+
+In order to understand how to use Attach Data, you will need a foundational knowledge on how Events flow in the SDK and the role of the SDK's Rules Engine.
+
+This document provides the information necessary to understand how to use Attach Data, and provides an example of how to use it.
+
+## SDK Events
+
+#### What is an Event?
+
+In the SDK, Events hold all of the data required by other SDK extensions to perform their necessary actions.
```

Comment:
- Please add "that is" between "data" and "required".

- Please replace "perform" with "complete'